src/jit/inline.h

```diff
@@ -13,6 +13,7 @@
 // InlineTarget -- enum, target of a particular observation
 // InlineImpact -- enum, impact of a particular observation
 // InlineObservation -- enum, facts observed when considering an inline
+// InlineContext -- class, remembers what inlines happened
```

Comment:
Would it make sense to group (and comment) these according to which can have multiple instances per candidate, which are one per-inline-candidate and which are per-compile (per-caller)?